mreg_cli/policy.py

```diff
@@ -445,7 +445,7 @@ def list_roles(args):
     callback=list_roles,
     flags=[
         Flag('name',
-             description='Role name filter (regexp)',
+             description='Role name, or part of name. You can use * as a wildcard.',
```

Comment:
The text here is a bit unclear. Do I need to use a wildcard to search for part of name? If I need to use a wildcard, the comment should be more like the one on line 310, if not, what purpose does the wildcard serve?